app/views/reviews/new.html.erb

```diff
@@ -12,7 +12,7 @@
       <%= label_tag "Title:" %>
       <%= text_field_tag :title %><br>
       <%= label_tag "Image URL:" %>
-      <%= text_field_tag :img_url %><br>
+      <%= text_field_tag :img_url, :default => "http://www.amandaspetcare.com/services/pet-sitting/dog-cat-grass.jpg" %><br>
```

Comment:
I hadnt realized I put this on the same branch. Not sure if I can take it out, it shouldnt affect anything. This is my start at user story 37, but I have a lot more changes to make.

Reply:
All good